packages/runtime/src/account.ts

```diff
@@ -184,6 +184,58 @@ export class StoreAccount implements StoreAccountI {
     return asset.definitions;
   }
 
+  /**
+   * Modifies Asset fields
+   * @param assetId Asset Index
+   * @param fields Fields for modification
+   */
+  modifyAsset (assetId: number, fields: AssetModFields): void {
+    const asset = this.getAssetDef(assetId);
+    if (asset === undefined) {
+      throw new TealError(ERRORS.ASA.ASSET_NOT_FOUND, { assetId: assetId });
+    }
+    // check for blank fields
+    if ((fields.reserve && asset.reserve === "") || (fields.freeze && asset.freeze === "") ||
+      (fields.clawback && asset.clawback === "")) {
+      throw new Error("Cannot reset a blank address");
```

Comment:
why? 
I think this should be correct. Example: What if `asset.reserve = ""` and we want to update `asset.freeze`? Then we will have `{reserve: "", freeze: "something else"}`

Reply:
Yes, in this idea I thought user will set reserve as undefined if he doesn't want to update.

I will update to use "" blank address